v2/passwordless/serverless/with-aws-lambda/session-verification.mdx

```diff
@@ -64,7 +59,68 @@ module.exports.handler = middy(verifySession(handler)).use(cors({
 //highlight-end
 ```
 
-## 4) Configure API Gateway
+
+</TabItem>
+<TabItem value="python">
+
+```python title="auth.py"
+import nest_asyncio # type: ignore
+nest_asyncio.apply() # type: ignore
+
+from fastapi import FastAPI
+from starlette.middleware.cors import CORSMiddleware
+from mangum import Mangum
+
+from supertokens_python import init, get_all_cors_headers
+from supertokens_python.framework.fastapi import get_middleware
+
+import config # type: ignore
+
+init(
+    supertokens_config=config.supertokens_config, # type: ignore
+    app_info=config.app_info, # type: ignore
+    framework=config.framework, # type: ignore
+    recipe_list=config.recipe_list, # type: ignore
+    mode="asgi",
+)
+
+app = FastAPI(title="SuperTokens Example")
+
+# highlight-start
+from fastapi import Depends
+from supertokens_python.recipe.session.framework.fastapi import verify_session
+from supertokens_python.recipe.session import SessionContainer
+
+@app.get("/user")
+def user(s: SessionContainer = Depends(verify_session())):
+    return {
+        "sessionHandle": s.get_handle(),
+        "userId": s.get_user_id(),
+        "accessTokenPayload": s.get_access_token_payload()
+    }
+# highlight-end
+
+app.add_middleware(get_middleware())
```

Comment:
this is not required here - cause this is the user's API

Reply:
I don't think so. How will SuperTokens provide values for SessionContainer that is being used in the `user()` function?